Integrations/integration-CounterTack.yml

```diff
@@ -0,0 +1,1716 @@
+
+
+    def add_behavior_tags():
+
+        """
+        Add specific tags to specified behavior
+
+        demisto parameter: (string) behavior_id
+            The unique ID of the behavior
+
+        demisto parameter: (Array) Body.
+            The tags to add to the behavior. seperate the tags with comma
+        """
+        behaviour_id = demisto.args().get('behaviour_id')
+        body = argToList(demisto.args().get('tags'))
+        if not isinstance(body, list):
+            body = [body]
+        response = behaviour_add_tags_request(behaviour_id,body)
+        behavior_tags = get_behavior_request(behaviour_id)
+
+        response = {
+            'tags':behavior_tags.get('tags'),
+            'behaviour_id':behaviour_id
+        }
+
+        context['CounterTack.Behavior(val.ID && val.ID === obj.ID)'] = response
+
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': response,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown("Behavior tags were added successfully", response),
+            'EntryContext': context
+        }
+        demisto.results(entry)
+
+    def delete_tags_behavior_request(behaviour_id,body):
+
+        url = '{}/behaviors/{}/tags'.format(BASE_PATH,behaviour_id)
+
+        response = http_request(
+            'DELETE',
+            url,
+            headers = GET_HEADERS,
+            body = json.dumps(body)
+        )
+        return response
+
+
+    def delete_behavior_tags():
+
+        """
+        Delete specific tags from behavior
+
+        demisto parameter: (string) behavior_id
+            The unique ID of the behavior
+
+        demisto parameter: (Array) Body.
+            The tags to delete from the behavior. seperate the tags with comma
+
+        """
+
+        behaviour_id = demisto.args().get('behaviour_id')
+        body = argToList(demisto.args().get('tags'))
+        if not isinstance(body, list):
+            body = [body]
+        response = delete_tags_behavior_request(behaviour_id,body)
+        response = get_behavior_request(behaviour_id)
+
+        response = {
+            'tags':response.get('tags'),
+            'behaviour_id':behaviour_id
+        }
+
+        context['CounterTack.Behavior(val.ID && val.ID === obj.ID)'] = response
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': response,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown("Endpoint tags were deleted successfully", response),
+            'EntryContext': context
+        }
+        demisto.results(entry)
+
+    """
+
+    FETCH INCIDENTS
+
+    """
+    def fetch_incidents():
+        now = datetime.utcnow()
+        last_run = demisto.getLastRun().get('time')
+        if last_run is None:
+            now = now - timedelta(days=3)
```

Comment:
the time delta to fetch from should be configurable and not 3 days by default, please add parameter for that